hirte.spec.in

```diff
@@ -156,6 +173,8 @@ This package contains the service controller command line tool.
 
 %install
 %meson_install
+install -p -D -m 0644 src/bindings/python/hirte/__init__.py %{buildroot}%{python3_sitelib}/%{name}/__init__.py
```

Comment:
I haven't packaged any python module via RPM yet, so I apologize upfront if the answer to my question is obvious :D
Shouldn't this two install lines here be done via `meson`?

Reply:
I don't believe, it's pure SPEC thing. @mwperina please help me here.

Reply:
Please do not mix python files with meson, I'd leave meson for C code and install python files using python spec macros:

https://docs.fedoraproject.org/en-US/packaging-guidelines/Python/#_example_spec_file

Reply:
meson can also be used to install python (see [here](https://mesonbuild.com/Python-module.html)). You mean meson and rpm build, right? 
Thanks for the link! That means it should be `%pyproject_install`, right? I am not sure what the two install lines do, but it seems like moving/copying files - thats why I asked about meson (and `%meson_install` would've taken care of it).

Reply:
I'd leave meson to build C, python doesn't need any special meson setup, everything for RPM could be handled using python specific RPM macros

Reply:
we'll need the `setup.py` for pypi, so I also like the idea of doing the C code with meson and keep leveraging `setup.py` for the python code. That's a lot more natural for python developers.

Reply:
We have setup.py already: https://github.com/dougsland/hirte/blob/rpm/src/bindings/python/setup.py#L1